### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by clemens-tolboom**
A quick and dirty Firefox developer workflow.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ More information coming soon.
 
 ## Configure your browser
 
+Make sure to run `grunt build` once before continue.
```
Comment (markhalliwell) [inline CONTRIBUTING.md:54]: Why do you need to do this?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -60,7 +62,11 @@ More information coming soon.
 
 ### Firefox
 
-(to be defined)
+(quick and dirty)
```
Comment (markhalliwell) [inline CONTRIBUTING.md]: I'd rather not say "quick and dirty". If this is the best option at the moment, then so be it. We can always update this later.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -60,7 +62,11 @@ More information coming soon.
...
+
+1. Check the current Dreditor greasemonkey location (about:addons)
+2. Right click Dreditor for install location
+3. Replace dreditor.user.js with a symlink to your development versions grunt build artifact `ln -s path-to-dreditor-dir/build/dreditor.js dreditor.user.js`
```
Comment (markhalliwell) [inline CONTRIBUTING.md:69]: `dreditor.user.js` isn't necessary, just `dreditor.js`

Comment (clemens-tolboom) [inline CONTRIBUTING.md:69]: As we take an installed plugin version of Dreditor as a starting point the `gm_scripts/config.xml` refers to `dreditor.user.js` that why I wrote that symlink.

I haven't tested `dreditor.js`. @markcarver did you?

Comment (markhalliwell) [inline CONTRIBUTING.md:69]: FF is an extension now, greasemonkey isn't officially supported for dreditor any longer. However for development purposes, we may have to. Let me think on this some more. 

Typos courtesy of Siri

**Comment by clemens-tolboom** [comment]
I've just installed 'Auto Refresh Plus' for Chrome on the `chrome://extensions/` page. Should we document this?

File: CONTRIBUTING.md (hunk 1 above)
Comment (clemens-tolboom) [inline CONTRIBUTING.md:54]: The build directory is not available before building once :)

**Comment by sun** [comment]
I agree with @markcarver - I don't think the initial `grunt build` addition is necessary.

For now, I wouldn't mark development via GreaseMonkey as deprecated, since it works excellently.  I'm using the identical symlink approach myself (which even works on Windows via `mklink`).  As long as Dreditor doesn't have to be installed as a browser extension, this is most likely the simplest and fastest way to contribute and work on Dreditor.

Here's how I've set up my development environment:
1. I followed the instructions on https://dreditor.org/development/contributing to install nodejs, npm, and grunt.
2. I executed `grunt` (as documented there) to perform a complete build.
3. I imported the resulting `build/dreditor.js` into Firefox/GreaseMonkey.
4. I deleted the file in the `gm_scripts/Dreditor` folder of my Firefox profile and replaced it with a symlink to `[dreditor-git-checkout/]build/dreditor.js`.

→ Dreditor is loaded directly from grunt's build directory + updated automatically.

To further improve this development flow, I created two PRs:
- #99 makes step (3) of above work correctly, so that the user script is still limited to drupal.org domains and not executed on all websites.
- #100 speeds up `grunt watch` for this development flow to rebuild/recompile the user script as quickly as possible.

**Comment by markhalliwell** [comment]
The docs at https://github.com/dreditor/dreditor/blob/1.x/CONTRIBUTING.md#firefox are pretty clear on how to auto install the built `xpi` extension in FF, not using GM userscripts. I'm definitely leaning towards this direction since we completely removed GM support. If we need to modify it further, please submit a new PR as I'm going to close this one.

**Comment by clemens-tolboom** [comment]
It's good to know when that's fixed https://github.com/dreditor/dreditor/commit/3f1f66432b55ecb8208a196987c57808f636012f

**Comment by sun** [comment]
@clemens-tolboom: Do you want to create an issue for your earlier remark?

> I've just installed 'Auto Refresh Plus' for Chrome on the chrome://extensions/ page. Should we document this?

Reloading/refreshing Dreditor always required me to restart the whole. effin. Chrome. browser in the past, so documenting every possible way to speed-up/improve that DX certainly seems to be a good idea ;)

**Comment by clemens-tolboom** [comment]
@sun I'm pretty of the dreditor dev right now. Will do when back.

**Pull request closed without merging**